README.md

```diff
@@ -63,16 +63,23 @@ t = Thread(target=client.listen)
 t.start()
 ```
 
-The client supports setting a custom generator that produces sequence numbers for the PDU packages. Per default a simple in memory generator is used which in conclusion is reset on (re)instantiation of the client, e.g. by an application restart. If you want to keep the sequence number to be persisted across restarts you can implement your own storage backed generator.
+A simple in memory generator that produces sequence numbers for the PDU packages is used, which is reset on (re)instantiation of the client, e.g. by an application restart. It starts by default with sequence number 0x00000001 but you may set the starting sequence yourself by declaring the generator beforehand and passing it to the Client:
+```python
+import smpplib.client
+
+generator = smpplib.client.SimpleSequenceGenerator(start_sequence=1234)
+client = smpplib.client.Client('example.com', SOMEPORTNUMBER, sequence_generator=generator)
+...
+```
 
-Example:
 
+The client supports setting a custom generator, so if you need more/different features associated with it, you can implement your own:
 ```python
 import smpplib.client
 
 import mymodule
 
-generator = mymodule.PersistentSequenceGenerator()
+generator = mymodule.MyAwesomeSequenceGenerator()
```

Comment:
Please elaborate a little about the interface such a Generator has to have

Reply:
If you don't have time for this, also please let us knowand @eigenein  will merge it as is.

Reply:
@podshumok @eigenein Feel free to add this info to the readme if you wish, I don't think I will have time to.